main.yml

```diff
@@ -766,6 +764,66 @@ service-registry:
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry
 
+
+## App Services (RHOAS) APIs
+app-services-apis:
+  title: Application Services
+  api:
+    subItems:
+      kafkamgmt:
+        title: "OpenShift Streams for Apache Kafka Management API"
+        api:
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: kas-fleet-manager.yaml?ref=doc-portal
+      connectormgmt:
+        title: "OpenShift Connectors Management API "
+        api:
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: connector_mgmt?ref=doc-portal
+      serviceregistrymgmt:
+        title: "OpenShift Service Registry Management API"
+        api:
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: srs-fleet-manager.json?ref=doc-portal
+      smarteventsmgmt:
+        title: "Smart Events Management API"
+        api:
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: smartevents-mgmt.yaml?ref=doc-portal
+      serviceaccountsmgmt:
+        title: "Service Accounts Management API"
+        api:
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: service-accounts.yaml?ref=doc-portal
+      kafkainstance:
+        title: "OpenShift Streams for Apache Kafka Instance API"
+        api:
+          readonly: true
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: kafka-admin-rest.yaml?ref=doc-portal
+      registryinstance:
+        title: "OpenShift Service Registry Instance API"
+        api:
+          readonly: true
```

Comment:
This is an config entry reserved for the future so we can disable "try it out" as described in

https://github.com/swagger-api/swagger-ui/issues/3725#issuecomment-904724595

Reply:
See: https://github.com/RedHatInsights/api-frontend/pull/470